bitnami/harbor/templates/exporter/exporter-dpl.yaml

```diff
@@ -100,13 +100,22 @@ spec:
             - name: HARBOR_DATABASE_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ include "harbor.core" . }}-envvars
+                  {{- if .Values.exporter.existingEnvVarsSecret }}
+                  name: {{ .Values.exporter.existingEnvVarsSecret }}
+                  {{- else }}
+                  name: {{ printf "%s-envvars" (include "harbor.core" .) }}
+                  {{- end }}
                   key: POSTGRESQL_PASSWORD
             - name: HARBOR_ADMIN_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ include "harbor.core" . }}-envvars
+                  {{- if .Values.exporter.existingEnvVarsSecret }}
+                  name: {{ .Values.exporter.existingEnvVarsSecret }}
+                  {{- else }}
+                  name: {{ printf "%s-envvars" (include "harbor.core" .) }}
+                  {{- end }}
                   key: HARBOR_ADMIN_PASSWORD
+                {{- end }}
```

Comment:
This line is causing the error. The `if` conditional is closed above, this `{{- end }}` line is unexpected.

Reply:
cheers

Reply:
Is there anything else I need to do or is this good to go ?